### ISSUE TITLE
Translation full documentation in other languages

### ISSUE THREAD

**Issue by webafrancois** [keepassxreboot/keepassxc#3267]
The only documentation is just a long FAQ

## Summary
I promote KeepassXC software to french users as a good option to save our passwords, because Free Open-Source software and many OS supported. But the only documentation is the FAQ or the quickstart in english. There is no long documentation, and the short is only in english.
So, it's not very friendly for us.
The sortcuts is only written in .md in the github repository.
https://github.com/keepassxreboot/keepassxc/blob/develop/docs/KEYBINDS.md
But not in the docs/FAQ ?

We can translate it in french, but many languages can be used too. It will be very usefull pour many people.

## Desired Behavior
Many translation of a full documentation structured in menu should be also very usefull.

## Possible Solution
Using some documentation like these : https://sts10.github.io/2017/06/27/keepassxc-setup-guide.html 
update it if possible and we can translate it.

**Comment by droidmonkey** [comment]
You can very easily use Google or Bing translating services to translate the whole web page. It is not perfect, but good enough for this purpose.

**Comment by droidmonkey** [comment]
Tada!! https://translate.googleusercontent.com/translate_c?depth=1&nv=1&rurl=translate.google.com&sl=auto&sp=nmt4&tl=fr&u=https://sts10.github.io/2017/06/27/keepassxc-setup-guide.html&xid=25657,15700022,15700186,15700190,15700256,15700259&usg=ALkJrhgqGqG4hYIedeGSTHz6vybwZsd4gA

**Comment by droidmonkey** [comment]
Tada!

https://www.translatetheweb.com/?from=&to=fr&a=https://sts10.github.io/2017/06/27/keepassxc-setup-guide.html

**Comment by webafrancois** [comment]
I'm sorry, but you don't understand that I wrote. I was proposing to have a translation of a complete documentation in the Keepassxc website in many languages. I don't need to translate this manual.
Actually, there is a too short setup guide on the site and new users are afraid by a so short documentation and not in their language.

**Comment by webafrancois** [comment]
KeepassXC software is translate in many languages : 
https://github.com/keepassxreboot/keepassxc/tree/develop/share/translations
But no manuals in their language for those people. 
For example, the new shortcuts array is only seeing in github. Not in the docs/FAQ.
https://github.com/keepassxreboot/keepassxc/tree/develop/docs

**Comment by droidmonkey** [comment]
We have full documentation, I am currently cleaning it up for our website. We are not going to make individual translations available, it is a monumental effort to do so. Web translation services are fee and effective.

**Comment by webafrancois** [comment]
> Web translation services are free and effective.

Do you say the same thing when you discover software in chinese ?
English is not the only language and if documentation exist in english, we can translate it.
But now, there isn't in your website. Only a big and confuse FAQ and a too short quickstart (wich is focused to the keepass browser integration) which is not very friendly for new users.
I suggest to integrate the manual of https://sts10.github.io/2017/06/27/keepassxc-setup-guide.html, update it and we can translate it in other language.

**Comment by droidmonkey** [comment]
I am looking into using Sphinx for our web-based documentation which can export to .po files that can be translated using Transifex.

**Comment by droidmonkey** [comment]
We will not be doing this, if anyone wants to translate our documentation then they are welcome to do so.
